protocol/scripts/deploys/mainnet/v2/core/99-core-post-deployment.ts

```diff
@@ -0,0 +1,81 @@
+import { ethers } from 'hardhat';
+import { 
+  ensureExpectedEnvvars,
+  mine
+} from '../../../helpers';
+import { 
+  connectToContracts,
+  getDeployedContracts
+} from '../contract-addresses';
+import { ITempleStrategy } from '../../../../../typechain';
+
+
+async function main() {
+  ensureExpectedEnvvars();
+
+  const [owner] = await ethers.getSigners();
+  const TEMPLE_V2_ADDRESSES = getDeployedContracts();
+  const TEMPLE_V2_INSTANCES = connectToContracts(owner);
+
+  // Setup Temple => dTemple & base strategy
+  {
+    // TRV can mint/burn $dTEMPLE
+    await mine(TEMPLE_V2_INSTANCES.TREASURY_RESERVES_VAULT.D_TEMPLE_TOKEN.addMinter(TEMPLE_V2_ADDRESSES.TREASURY_RESERVES_VAULT.ADDRESS));
+
+    const debtCeiling: ITempleStrategy.AssetBalanceStruct[] = [{
+        asset: TEMPLE_V2_ADDRESSES.CORE.TEMPLE_TOKEN,
+        balance: ethers.utils.parseEther("100000"),
```

Comment:
FYI only -- It's fine as is for now, but at some point we'll have to pull out all the decisions to be made for production deployment.

These debt ceilings/etc aren't confirmed yet, i just made up numbers for testnet

Reply:
Added TODO comment, should we add this somewhere else to remember?

Reply:
It's spread across the files...I think we can build up a google sheets of all of the things, and then use that to go through with Lux & Pengu to get consensus. 

We can do this *after* the audit fixes though, as things might change slightly